src/platform/nRF5/args.gni

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+print("################################################################")
```

Comment:
No console spam during innocent builds please. Everyone who builds everything in VSCode will see this.

Reply:
Sure. Thanks.